inlong-sdk/sort-sdk/src/main/java/org/apache/inlong/sdk/sort/impl/decode/MessageDeserializer.java

```diff
@@ -48,12 +55,20 @@ public class MessageDeserializer implements Deserializer {
     private static final String INLONG_GROUPID_KEY = "inlongGroupId";
     private static final String INLONG_STREAMID_KEY = "inlongStreamId";
 
+    private static final String INLONGMSG_ATTR_STREAM_ID = "streamId";
+    private static final String INLONGMSG_ATTR_GROUP_ID = "groupId";
+    private static final String INLONGMSG_ATTR_TIME_T = "t";
+    private static final String INLONGMSG_ATTR_TIME_DT = "dt";
+    private static final String INLONG_ATTR_SOURCE_IP = "srcIp";
+    private static final char INLONGMSG_ATTR_ENTRY_DELIMITER = '&';
+    private static final char INLONGMSG_ATTR_KV_DELIMITER = '=';
+
     public MessageDeserializer() {
     }
 
     @Override
     public List<InLongMessage> deserialize(ClientContext context, InLongTopic inLongTopic, Map<String, String> headers,
-            byte[] data) throws Exception {
+                                           byte[] data) throws Exception {
```

Comment:
It is not necessary to indent with the above parameters.

Reply:
fixed, thx